Update description for VariableDeclarationUsageDistance

I have read check documentation: https://checkstyle.org/checks/xxxxxx/nameofaffectedcheck.html
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words


```
$ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
        <module name="VariableDeclarationUsageDistance">
            <property name="allowedDistance" value="1" />
        </module>
    </module>
</module>

$ cat Main.java 
/** some javadoc. */
public class Main {

  /**
   * some javadoc.
   */
  public static void main(String[] args) {
    int pi = 0;                                                                 // declare
    int square = 32;
    int root = 8;
    int[] profit = {1, 0, -1};
    System.out.println("Starting calculation...");
    System.out.println("Iterating...");
    String message = "flow";
    pi = 2;                                                                       // first usage 
    int currentIndex = 0;
    for (int i = 1; i <= 5; i++) {
      currentIndex += 1;
    }
    System.out.println("Sum: ");
  }
}

$ java -jar testing/checkstyle-10.21.4-all.jar -c config.xml Main.java 
Starting audit...
[ERROR] /mnt/5D92528E6B945467/test/Main.java:8:5: Distance between variable 'pi' declaration
 and its first usage is 3, but allowed 1.  Consider making that variable final if you still need to
 store its value in advance (before method calls that might have side effects on the original value). 
[VariableDeclarationUsageDistance]
Audit done.
Checkstyle ends with 1 errors.
```


**Problem.**
Correct distance between `pi`'s declaration and first usage is 6. for some reason `square`, `root`, `message` variables are not accounted in the calculation. try to remove them or declare additional variables, you will get the same distance value.

--- a/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java
+++ b/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java
@@ -37,7 +37,7 @@ import com.puppycrawl.tools.checkstyle.utils.TokenUtil;
 /**
  * <div>
  * Checks the distance between declaration of variable and its first usage.
- * Note : Variable declaration/initialization statements are not counted while calculating length.
+ * Note: Variable declaration/initialization statements are not counted while calculating length.
  * </div>
  * <ul>
  * <li>

--- a/src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/VariableDeclarationUsageDistanceCheck.xml
+++ b/src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/VariableDeclarationUsageDistanceCheck.xml
@@ -6,7 +6,7 @@
              parent="com.puppycrawl.tools.checkstyle.TreeWalker">
          <description>&lt;div&gt;
  Checks the distance between declaration of variable and its first usage.
- Note : Variable declaration/initialization statements are not counted while calculating length.
+ Note: Variable declaration/initialization statements are not counted while calculating length.
  &lt;/div&gt;</description>
          <properties>
             <property default-value="3" name="allowedDistance" type="int">

--- a/src/site/xdoc/checks/coding/variabledeclarationusagedistance.xml
+++ b/src/site/xdoc/checks/coding/variabledeclarationusagedistance.xml
@@ -11,7 +11,7 @@
       <subsection name="Description" id="Description">
         <div>
           Checks the distance between declaration of variable and its first usage.
-          Note : Variable declaration/initialization statements are not counted
+          Note: Variable declaration/initialization statements are not counted
           while calculating length.
         </div>
       </subsection>

--- a/src/site/xdoc/checks/coding/variabledeclarationusagedistance.xml.template
+++ b/src/site/xdoc/checks/coding/variabledeclarationusagedistance.xml.template
@@ -11,7 +11,7 @@
       <subsection name="Description" id="Description">
         <div>
           Checks the distance between declaration of variable and its first usage.
-          Note : Variable declaration/initialization statements are not counted
+          Note: Variable declaration/initialization statements are not counted
           while calculating length.
         </div>
       </subsection>

